app/src/ui/diff/side-by-side-diff-row.tsx

```diff
@@ -548,6 +599,7 @@ export class SideBySideDiffRow extends React.Component<
 
   private onClickHunk = () => {
     if (this.props.hideWhitespaceInDiff) {
+      this.setState({ showWhitespaceHint: DiffColumn.Before })
```

Comment:
Not sure how hard is it to fix this, but if the hunk is all in the `After` column, the popover looks a bit weird 😅 

![image](https://user-images.githubusercontent.com/1083228/134375015-fd1de247-98be-4849-8de0-993bc79bdb29.png)

Reply:
Agreed, that does look weird. I've updated the logic to prefer the left hand side except in the case of added lines like the one above in https://github.com/desktop/desktop/pull/12979/commits/e8d57a25e5b56c31a6e9b249c89d1b4d18d486c8